woocommerce/woocommerce
# [Settings] Update feature flag usage

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





This PR moves the Settings feature flag into the correct folder and removes a check for an option value to turn the feature on. Simply updating the config will be sufficient. This is helpful so switching branches from `trunk` to any development of Settings won't require a `composer install`.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Locally, update the development config value
```diff
--- a/plugins/woocommerce/client/admin/config/development.json
+++ b/plugins/woocommerce/client/admin/config/development.json
@@ -26,7 +26,7 @@
                "printful": true,
                "remote-inbox-notifications": true,
                "remote-free-extensions": true,
-               "settings": false,
+               "settings": true,
                "shipping-label-banner": true,
                "subscriptions": true,
                "store-alerts": true,
```
2. We've renamed a file, so be sure to update composer. `cd plugins/woocommerce && composer update`
3. `pnpm --filter='@woocommerce/plugin-woocommerce' watch:build`
4. WooCommerce > Settings
5. A placeholder for a new experience should render



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

Change to underlying Settings feature usage

</details>
